container/crio/handler.go

```diff
@@ -148,13 +148,22 @@ func newCrioContainerHandler(
 		Namespace: CrioNamespace,
 	}
 
-	// Find out if we need network metrics reported for this container.
-	// Containers that don't have their own network -- this includes
-	// containers running in Kubernetes pods that use the network of the
-	// infrastructure container -- does not need their stats to be
-	// reported. This stops metrics being reported multiple times for each
-	// container in a pod.
-	metrics := common.RemoveNetMetrics(includedMetrics, cInfo.Labels["io.kubernetes.container.name"] != "POD")
+	nonNetworkMetrics := common.RemoveNetMetrics(includedMetrics, true)
```

Comment:
While at it, you may remove the second argument of `RemoveNetMetrics` now.

Reply:
it looks like containerd/docker use them